src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/iscsi/iscsi.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, OnInit } from '@angular/core';
+import { CephShortVersionPipe } from '../../../shared/pipes/ceph-short-version.pipe';
+import { DimlessBinaryPipe } from '../../../shared/pipes/dimless-binary.pipe';
+import { DimlessPipe } from '../../../shared/pipes/dimless.pipe';
+import { ListPipe } from '../../../shared/pipes/list.pipe';
+import { RelativeDatePipe } from '../../../shared/pipes/relative-date.pipe';
+import { TcmuIscsiService } from '../../../shared/services/tcmu-iscsi.service';
+
+@Component({
+  selector: 'cd-iscsi',
+  templateUrl: './iscsi.component.html',
+  styleUrls: ['./iscsi.component.scss']
+})
+export class IscsiComponent implements OnInit {
+
+  daemons: Array<any> = null;
```

Comment:
Doesn't ``daemons: Array<any> = [];`` make more sense here?

Reply:
I also prefer to have an empty list, this way the table can be initialized properly.

`daemons = [];` will do the same.